### PR TITLE
`/libs/model` test type fixes

### PR DESCRIPTION
**Pull request by timolegros**
## Link to Issue
Closes: #7540

### CODE REVIEW

**Comment by timolegros** [comment]
The PR is ready but leaving in draft until #7517 is merged.

**Comment by Rotorsoft** [comment]
LGTM so far... will approve when ready for review

**Review by rbennettcw — APPROVED** [review]

**Review by Rotorsoft — APPROVED** [review]

**Pull request merged**
